test/integration/users.test.js

```diff
@@ -1040,4 +1037,61 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    beforeEach(async function () {
+      superUserId = await addUser(superUser);
+      superUserAuthToken = authService.generateAuthToken({ userId: superUserId });
+    });
```

Comment:
We already are adding a superUser and generating superUserAuthToken in outer beforeEach block so this is not required

https://github.com/Real-Dev-Squad/website-backend/blob/f61ff709973a586dda17eacaa0b9fef805de72e4/test/integration/users.test.js#L32-L37

Reply:
Okay I'll remove that.